LL/h_osráin.ttl

```diff
@@ -209,15 +209,23 @@
     foaf:name "Linga";
     rel:childOf <#Mail-b265b280>.
 
+# In Sil_Birn, Daiglinguae is one person but <#Daig> 
+  and <#Linga> are presented as separate names here. 
+  If <#Daig> and <#Linga> are separate people,
+  then, in each section, Mál has 8 sons, not 
+  7, as per the text on each occasion. I have
+  therefore linked them all together via owl:sameAs. EPT  
+

```

Comment:
Each one of the above lines needs a # to mark it as a comment. Could you add those from line 213-217.